python/mujinplanningclient/realtimerobotplanningclient.py

```diff
@@ -386,1649 +740,3318 @@ def GetTransform(self, targetname, connectedBodyName='', linkName='', geometryNa
+        # type: (float, Optional[types.SetRobotBridgeResumeParametersDynamicEnvironmentState], Optional[int], str, Optional[str], Optional[str], Optional[types.SetRobotBridgeResumeParametersRobotBridgeConnectionInfo], Optional[list[types.SetRobotBridgeResumeParametersLocationCollisionInfosArrayElement]], Optional[Any]) -> Optional[Any]
         """
         Args:
-            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 10)
-            unit (str, optional): The unit of the given values. (Default: 'mm')
-            robotname (str, optional): Name of the robot
-            toolname (str, optional): Name of the manipulator. Defaults to currently selected tool
-            robotBridgeConnectionInfo (dict, optional): Information to set up a client to the robot bridge.
-            locationCollisionInfos (dict, optional): List of external collision IOs to be computed and sent in realtime.
+            timeout: Time in seconds after which the command is assumed to have failed. (Default: 10)
+            dynamicEnvironmentState: Dynamic environment state that allows the user to set/create objects in a particular state dynamically. (Default: None)
+            debuglevel: Sets the debug level for the planning logs. For development. 3=INFO, 4=DEBUG, 5=VERBOSE. (Default: None)
+            unit: The unit of the given values. (Default: 'mm')
+            robotname: Name of the robot (Default: None)
+            toolname: Name of the manipulator. Defaults to currently selected tool (Default: None)
+            robotBridgeConnectionInfo: Information to set up a client to the robot bridge. (Default: None)
+            locationCollisionInfos: List of external collision IOs to be computed and sent in realtime. (Default: None)
         """
         taskparameters = {
             'command': 'SetRobotBridgeResume',
-        }
+            'unit': unit,
+        }  # type: dict[str, Any]
+        if dynamicEnvironmentState is not None:
+            taskparameters['dynamicEnvironmentState'] = dynamicEnvironmentState
+        if debuglevel is not None:
+            taskparameters['debuglevel'] = debuglevel
+        if robotname is not None:
+            taskparameters['robotname'] = robotname
+        if toolname is not None:
+            taskparameters['toolname'] = toolname
+        if robotBridgeConnectionInfo is not None:
+            taskparameters['robotBridgeConnectionInfo'] = robotBridgeConnectionInfo
+        if locationCollisionInfos is not None:
+            taskparameters['locationCollisionInfos'] = locationCollisionInfos
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, timeout=timeout)
 
-    def SetRobotBridgeServoOn(self, servoon, robotname=None, timeout=3, fireandforget=False):
+    def SetRobotBridgeServoOn(self, isservoon, robotname=None, timeout=3, fireandforget=False, dynamicEnvironmentState=None, debuglevel=None):
```

Comment:
parameter name changed

Reply:
Should be fixed. https://github.com/mujin/mujinplanningclientpy/pull/45/commits/ac88d2b414c132b133e279f100e321e5a41ce213